### PR TITLE
PUB-1237 - Flyway Reference Data Load

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1237
https://tools.hmcts.net/jira/browse/PUB-1190

### Change description ###

Add in Flyway reference data load, to load in the court details. This relates to CR: CHG5007525

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by alberto-hortiguela — APPROVED** [review]

File: charts/pip-data-management/values.yaml [hunk 1]
```diff
@@ -26,3 +26,4 @@ java:
   #        - name: AppInsightsInstrumentationKey
   #          alias: azure.application-insights.instrumentation-key
   environment:
+    RUN_DB_MIGRATION_ON_STARTUP: true
```
Comment (timja) [inline charts/pip-data-management/values.yaml:29]: this shouldn't be enabled by default in values.yaml only in values.preview.template.yaml

**Review by timja — CHANGES_REQUESTED** [review]

**Review by alberto-hortiguela — APPROVED** [review]

**Review by timja — APPROVED** [review]

File: infrastructure/variables.tf [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
 variable "common_tags" {
-  type = "map"
+  type = map(string)
+}
+variable "team_name" {
+  default = "PIP DevOps"
 }
+variable "team_contact" {
+  default = "#vh-devops"
+}
```
Comment (timja) [inline infrastructure/variables.tf:26]: what are these here for?

File: infrastructure/main.tf [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
   features {}
 }
+
+data "azurerm_resource_group" "example" {
+  name = "pip-ss-sbox-rg"
+}
+
+output "id" {
+  value = data.azurerm_resource_group.example.id
+}
```
Comment (timja) [inline infrastructure/main.tf:11]: what is all this for?

File: infrastructure/variables.tf (hunk 2 above)
Comment (chris-pateman) [reply infrastructure/variables.tf:26]: Have them in the other infra pipelines, so copied here.
do they get populated by the pipeline?

File: infrastructure/main.tf (hunk 3 above)
Comment (chris-pateman) [reply infrastructure/main.tf:11]: not needed, was part of testing. will remove

File: infrastructure/variables.tf (hunk 2 above)
Comment (timja) [reply infrastructure/variables.tf:26]: yes https://github.com/hmcts/cnp-jenkins-library/blob/master/vars/spinInfra.groovy#L50

Comment (chris-pateman) [reply infrastructure/variables.tf]: removed

**Pull request merged**